pkg/daemon/ceph/client/filesystem.go

```diff
@@ -335,10 +335,15 @@ func RemoveFilesystem(context *clusterd.Context, clusterName, fsName string) err
 		return fmt.Errorf("Failed to delete ceph fs %s. err=%+v", fsName, err)
 	}
 
-	err = deleteFSPools(context, clusterName, fs)
-	if err != nil {
-		return fmt.Errorf("failed to delete fs %s pools. %+v", fsName, err)
+	if !preservePools {
+		err = deleteFSPools(context, clusterName, fs)
+		if err != nil {
+			return fmt.Errorf("failed to delete fs %s pools. %+v", fsName, err)
+		}
+	} else {
+		logger.Infof("<PreservePools> is set in filesystem <%s>. Pools not deleted", fsName)
```

Comment:
I'd suggest not including the brackets in logging messages. We don't have a current pattern of that in rook. It's readable without the brackets.